### PR TITLE
Restore ability to use default export in CommonJS

### PR DESCRIPTION
**Pull request by sindresorhus**
So you can use `require('is')` instead of `require('is').default`.

It's a hack, but it seems to work fine.

### CODE REVIEW

**Comment by sindresorhus** [comment]
// @ltetzlaff

**Comment by SamVerschueren** [comment]
Sweet, never thought about this :)!

File: source/index.ts [hunk 1]
```diff
@@ -247,3 +247,7 @@ Object.defineProperties(is, {
 });
 
 export default is; // tslint:disable-line:no-default-export
+
+// For CommonJS default export support
+module.exports = is;
+module.exports.default = is;
```
Comment (kevva) [inline source/index.ts]: Doesn't `export default is` get compiled to this?

Comment (sindresorhus) [reply source/index.ts]: No, it get's compiled to `exports.default = is`...

Comment (kevva) [reply source/index.ts]: Ah, too bad.

**Comment by ltetzlaff** [comment]
Cool, did not think of that either 👍

**Pull request merged**